hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/TraceabilitySuite.java

```diff
@@ -4225,7 +4272,7 @@ HapiSpec traceabilityE2EScenario16() {
                                                     .setGas(79000)
                                                     .setRecipientContract(
                                                             spec.registry().getContractId(PRECOMPILE_CALLER))
-                                                    .setGasUsed(5232)
+                                                    .setGasUsed(5330)
```

Comment:
Do we know what caused this discrepancy?  (Or is this two of those off-by-ones together?)

Reply:
Not exactly but it seems we used about 100 more gas?